user/views.py

```diff
@@ -48,3 +48,24 @@ def post(self, request):
         except KeyError:
             return JsonResponse({"message": "INVALID_KEYS"}, status = 400)
 
+class SignInView(View):
+    def post(self, request):
+        data = json.loads(request.body)
+
+        try:
+           if User.objects.filter(login_id = data['login_id']).exists():
+               user = User.objects.get(login_id = data['login_id'])
+
+               if bcrypt.checkpw(data['password'].encode('utf-8'), user.password.encode('utf-8')):
+
+                   token = jwt.encode({'login_id' : user.login_id}, SECRET_KEY, algorithm = 'HS256')
+                   token = token.decode('utf-8')
+
+                   return JsonResponse({"token" : token}, status = 200)
```

Comment:
token변수에 두번 대입해서 처리해도 되지만 62번 라인은 삭제해도 되고 63번 공백은 불필요하여 라인수도 줄이기 위하여 하기와 같이 넘겨줄때 디코딩해서 처리해주세요. 
```suggestion
                   return JsonResponse({"token" : decode('utf-8')}, status = 200)
```